src/components/about/Content.js

```diff
@@ -0,0 +1,73 @@
+import { Divider, Flex, Heading, Text, theme } from '@chakra-ui/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * @component
+ * @example
+ * return (
+ *   <Content
+ *     heading="MISSION"
+ *     message="Our mission statement..."
+ *     marginTop="25px"
+ *     marginBottom="25px"
+ *     dividerMargin="10px"
+ *   />
+ * )
+ */
+const Content = ({
+  heading,
+  message,
+  marginTop,
+  marginBottom,
+  dividerMargin,
+}) => {
+  return (
+    <Flex
+      direction={['column', 'column', 'column', 'row', 'row']}
+      marginTop={marginTop}
+      marginBottom={marginBottom}
+    >
+      <Heading
+        as="h2"
+        lineHeight="1"
+        verticalAlign="top"
+        fontFamily={theme.fonts['heading-slab']}
+        color={theme.colors['rbb-black-200']}
+        textAlign={['center', 'center', 'center', 'right']}
+        minW="285px"
+        paddingRight={['0', '0', '22px']}
+      >
+        {heading}
+      </Heading>
+      <Divider
+        marginLeft={[dividerMargin, dividerMargin, '3px', '3px']}
+        marginRight={[dividerMargin, dividerMargin, '28px', '28px']}
+        marginTop={['25px', '25px', '25px', '0']}
+        marginBottom={['25px', '25px', '25px', '0']}
+        border="1px solid"
+        borderColor="#BA2A2A"
```

Comment:
Can leverage `theme.colors['rbb-red']` here

Reply:
chakra for some reason does not like this specifically to the borderColor. Giving it a variable value doesn't set any borderColor at all, not sure why since giving it the manual hex value is fine. Does anyone know how to implement this?

Reply:
@juanri0s that's super strange. Don't want this to hold you up, just throw it the hex value for now and we can refactor later!